src/bridge.ts

```diff
@@ -199,6 +200,9 @@ class DenoBridge {
       env.DENO_DIR = this.denoDir
     }
 
+    // Ensure PATH is always set as otherwise we are not able to find the global deno binary
+    env[pathKey()] = inputEnv[pathKey({ env: inputEnv })] || process.env[pathKey()]
```

Comment:
Doesn't `pathKey` always need the environment variables passed in?

If I look at [this](https://github.com/sindresorhus/path-key/blob/main/index.js#L11) correctly, and if the variable happens to be `PATH` on Windows, you're then setting it as `Path` in the final env object?

Or am I misunderstanding? 🤔

Reply:
It sets path to whatever the `PATH` casing is in `process.env`.
So if someone passes in PATH, ti will be converted to `Path` on windows, or whatever the current casing is.